killbill-integration-tests/plugins/avatax/test_tax.rb

```diff
@@ -1,15 +1,34 @@
 $LOAD_PATH.unshift File.expand_path('../../..', __FILE__)
+$LOAD_PATH.unshift File.expand_path('../..', __FILE__)
 
-require 'test_base'
+require 'plugin_base'
+require 'avatax/client'
 
 module KillBillIntegrationTests
 
-  class TestTax < Base
+  class TestTax < KillBillIntegrationTests::PluginBase
```

Comment:
Is this test passing for you? I'm getting invoice check errors, which look like as if there was no call made to Avatax.